texastribune/data-visuals-create
# Landing page

**Changes:**
- Turns `index.html` into a landing page, moves the templates for JS and static graphics into the `templates/` folder — we're making this change because including templates for graphics at the top level by default -> a lot of unused graphics littered in the graphics plugin. The templates are named `graphic.html` and `graphic-static.html`.
-  Checks whether the HTML template includes a JS pack in `base.html` (our new landing page does not include any JS by default)

**Lingering todos:**
- Maybe have some placeholder text directly in the HTML `graphic.html` and `graphic-static.html`, otherwise they're just blank pages before we run `npm run data:fetch` <-- **Out of scope, will address in a later PR**
- Should the intro text be in a Google Doc, instead of hardcoded in the HTML? It's another doc we'd need to fetch in the config file <-- **If it's not hardcoded, we'd need to fetch it**
- Update CHANGELOG
- Bump version

**To test**
- Check out `landing-page` and run your/path/to/data-visuals-create/bin/data-visuals-create graphic my-test-graphic.
- Run `npm run build`. Check that no metadata was generated in `manifest.json` in the `dist/` folder.
- Now copy `graphic.html` and/or `graphic-static.html` into the `app/` folder and name the files something else. (We usually rename our templates.)
- Run `npm run build`. Check that metadata was generated for those two graphics.
